cfme/tests/intelligence/reports/test_reports.py

```diff
@@ -105,3 +115,36 @@ def test_reports_custom_tags(appliance, request, create_custom_tag):
     report = appliance.collections.reports.create(**report_data)
     request.addfinalizer(report.delete)
     assert report.exists
+
+
+@test_requirements.report
+@pytest.mark.tier(0)
+@pytest.mark.ignore_stream("5.10")
+@pytest.mark.parametrize("based_on", REPORT_BASED_ON)
```

Comment:
Q: why parametrize the test case in this way? I.e. why not just use `REPORT_BASED_ON` in the test case? Just curious

Reply:
Um, no reason in particular actually. Earlier I was planning on using a dictionary, so I had parametrized it that way, and I guess I forgot to change that later. But better to change it, I guess.